src/components/Loading/Loading.styles.js

```diff
@@ -0,0 +1,30 @@
+import { css, keyframes } from '@emotion/core';
+import { Radius, Space } from 'lib/theme';
+
+const spin = keyframes({
+  to: { transform: 'rotate(360deg)' },
+});
+
+const styles = {
+  loader: css({
+    width: 30,
+    height: 30,
+    border: '4px solid #ffeeba',
+    borderRadius: '50%',
+    borderTopColor: '#856404',
+    borderRightColor: '#856404',
+    animation: `${spin} 1s linear infinite`,
+    marginRight: Space.S30,
+  }),
+  root: css({
+    background: '#fff3cd',
+    border: `1px solid #ffeeba`,
+    borderRadius: Radius.ROUNDED,
+    color: '#856404',
+    padding: Space.S20,
+    display: 'flex',
+    alignItems: 'center',
```

Comment:
Can the colors in here be moved into theme?

Reply:
I left these in the styles file because they aren't colors that we will be using in the future (I don't think, anyways). With that in mind should they still be moved to the colors file?

Reply:
Good call. How about moving them to a constant in this file with a stupid name like `const colorsSpecificToThisFile`?